api/agentpb/agent.proto

```diff
@@ -139,24 +139,32 @@ message StartActionRequest {
     string dsn = 1;
     string query = 2;
     MysqlExplainOutputFormat output_format = 3;
+    // Contains files and their contents which can be used in DSN.
+    TextFiles text_files = 4;
   }
   // MySQLShowCreateTableParams describes MySQL SHOW CREATE TABLE action parameters.
   message MySQLShowCreateTableParams {
     // DSN for the service. May contain connection (dial) timeout.
     string dsn = 1;
     string table = 2;
+    // Contains files and their contents which can be used in DSN.
+    TextFiles text_files = 3;
   }
   // MySQLShowTableStatusParams describes MySQL SHOW TABLE STATUS action parameters.
   message MySQLShowTableStatusParams {
     // DSN for the service. May contain connection (dial) timeout.
     string dsn = 1;
     string table = 2;
+    // Contains files and their contents which can be used in DSN.
+    TextFiles text_files = 3;
   }
   // MySQLShowIndexParams describes MySQL SHOW INDEX action parameters.
   message MySQLShowIndexParams {
     // DSN for the service. May contain connection (dial) timeout.
     string dsn = 1;
     string table = 2;
+    // Contains files and their contents which can be used in DSN.
+    TextFiles text_files = 3;
```

Comment:
Maybe better to call it tls_files? This could give a more understanding that inside certificates but not plain text files.
What do you think?

Reply:
You want to change name everywhere, or just for all actions for mysql? I chose this name because we already using it on different places and in frame of compatibility I chose same name for this field in actions too.

Reply:
Probably only here, it can be a breaking changes

Reply:
Done